scripts/npm/link/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var path = require('path'),
+    fs = require('fs'),
+    componentsDir = path.join(__dirname, '..', '..', '..', 'src'),
+    components = getDirectories( componentsDir ),
+    exec = require( 'child-process-promise' ).exec;
```

Comment:
ESLint will fail this, I'd follow the linter and let the other places fail till they're updated.